### PR TITLE
Fixing bug #67: wrap console.log

### PR DESCRIPTION
**Pull request by AtilMohAmine**
Running JavaScript in the iframe.

Screenshot from 2022-08-26 17-31-51

### CODE REVIEW

File: app/javascript/slices/terminalSlice.js [hunk 1]
```diff
@@ -7,14 +8,24 @@ export const runCode = createAsyncThunk(
...
         try {
+          const internalLogs = [];
+          // eslint-disable-next-line func-names
+          iframe.contentWindow.console.log = function (value) {
+            internalLogs.push(value);
+          };
+
           // eslint-disable-next-line no-eval
-          result = eval(code);
+          iframe.contentWindow.eval(code);
```
Comment (IxxyDev) [inline app/javascript/slices/terminalSlice.js:23]: No way to get rid of .eval()? I see, that it was used before your commit but...

This method is very insecure.

Comment (AtilMohAmine) [reply app/javascript/slices/terminalSlice.js:23]: Unfortunately there is no alternative to use in client side.

Comment (IxxyDev) [reply app/javascript/slices/terminalSlice.js:23]: And this "long tab" in World25 line is assumed to be there?

Comment (AtilMohAmine) [reply app/javascript/slices/terminalSlice.js:23]: Yes, iframe must be deleted after execution.

Comment (IxxyDev) [reply app/javascript/slices/terminalSlice.js:23]: LGTM

**Comment by mikhaylov-ya** [comment]
Нашел интересную либу https://github.com/patriksimek/vm2, вроде должна зайти. Там можно сконфигурировать доступную пользователю часть стандартной библиотеки и документация неплохая. Как думаешь, получится ее использовать?

**Comment by dzencot** [comment]
eval replaced, please check https://github.com/Hexlet/hexlet-editor/pull/85

**Pull request closed without merging**